pygmt/datasets/samples.py

```diff
@@ -125,39 +125,23 @@ def _load_japan_quakes():
     )
 
 
-def load_ocean_ridge_points(**kwargs):
+def _load_ocean_ridge_points():
     """
-    (Deprecated) Load a table of ocean ridge points for the entire world as a
+    Load a table of ocean ridge points for the entire world as a
     pandas.DataFrame.
 
-    .. warning:: Deprecated since v0.6.0. This function has been replaced with
-       ``load_sample_data(name="ocean_ridge_points")`` and will be removed in
-       v0.9.0.
-
-    This is the ``@ridge.txt`` dataset used in the GMT tutorials.
-
-    The data are downloaded to a cache directory (usually ``~/.gmt/cache``) the
-    first time you invoke this function. Afterwards, it will load the data from
-    the cache. So you'll need an internet connection the first time around.
-
     Returns
     -------
     data : pandas.DataFrame
-        The data table. Columns are longitude and latitude.
+        The data table. The column names are "longitude" and "latitude".
     """
-
-    if "suppress_warning" not in kwargs:
-        warnings.warn(
-            "This function has been deprecated since v0.6.0 and will be removed "
-            "in v0.9.0. Please use load_sample_data(name='ocean_ridge_points') "
-            "instead.",
-            category=FutureWarning,
-            stacklevel=2,
-        )
-
     fname = which("@ridge.txt", download="c")
     data = pd.read_csv(
```

Comment:
The two lines:
```
data = read_csv(xxx)
return data
```
can be shortened to `return read_csv(xxx)`.

```suggestion
    return pd.read_csv(
```